usrsctplib/user_recv_thread.c

```diff
@@ -346,7 +349,7 @@ recv_function_raw(void *arg)
 		msg.msg_iovlen = MAXLEN_MBUF_CHAIN;
 		msg.msg_control = NULL;
 		msg.msg_controllen = 0;
-		ncounter = n = recvmsg(SCTP_BASE_VAR(userspace_rawsctp), &msg, 0);
+		ncounter = n = (int)recvmsg(SCTP_BASE_VAR(userspace_rawsctp), &msg, 0);
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/a8adc4ae7cdfd014637ae3c0caf55d2bfda5ffc5